scripts/docker/keycloak/master-realm.json

```diff
@@ -101,9 +101,9 @@
       "attributes" : { }
     } ],
     "client" : {
+      "ort-server-ui-dev" : [ ],
```

Comment:
The commit message is a little bit misleading, because there are also client scopes in Keycloak. Maybe `Rename the client for ....`? The same for the next commit.

Reply:
In confused "scope" with "client", also in the the following commit message.